.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: local
+    hooks:
+      - id: mojo-format
+        name: mojo-format
+        entry: mojo format -l 80
```

Comment:
**Suggestion** Remove the `-l 80` as `mojo format` properly defaults to 80 characters line length now.  See https://github.com/modularml/mojo/pull/2244.

Reply:
Ok, sure. Just noting that it seems we're deviating from black's default (88 chars per line) I hope it's not going to annoy people who migrate their python code to mojo in the future.

Reply:
I agree. We can lazily evaluate that in the future once the language and tools mature.  Right now we're quite a ways from the direct migration of Python to Mojo.